src/__experimental__/components/simple-color-picker/simple-color-picker.component.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ColorOption from './color-option/color-option.component.js';
+import tagComponent from '../../../utils/helpers/tags/tags';
+import { StyledSimpleColorPicker, StyledColorOptions } from './simple-color-picker.style';
+import guid from '../../../utils/helpers/guid';
+
+class SimpleColorPicker extends React.Component {
+  isOptionChecked(color) {
+    return this.props.selectedColor === color;
+  }
+
+  getSingleOption(color) {
+    const isChecked = this.isOptionChecked(color);
+    const optionId = guid();
+
+    return (
+      <ColorOption
+        name={ this.props.name }
+        onChange={ this.props.onChange }
+        color={ color }
+        checked={ isChecked }
+        key={ optionId }
+        optionId={ optionId }
+      />
+    );
+  }
+
+  getColorOptions() {
+    return this.props.availableColors.map(color => this.getSingleOption(color));
+  }
+
+  render() {
+    return (
+      <StyledSimpleColorPicker { ...tagComponent('simple-color-picker', this.props) }>
+        <StyledColorOptions aria-label={ this.props.name } role='radiogroup'>
+          {this.getColorOptions()}
+        </StyledColorOptions>
+      </StyledSimpleColorPicker>
+    );
+  }
+}
+
+SimpleColorPicker.propTypes = {
+  /** An array of color choices to display. */
+  availableColors: PropTypes.array,
```

Comment:
shouldn't that be defined in the `OptionsHelper`?

Reply:
same as in color option, colors are up to the user